### ISSUE TITLE
Training train_2d_transferoemer-v5. py error

### ISSUE THREAD

**Issue by mengW6** [Garfield-kh/TM2D#16]
When running train_2d_transformar_v5. py, the error shown in the figure below appears. After searching, it seems that the index label is out of range. Have you encountered this problem before? Can you help me figure out how to modify it?
![image](https://github.com/user-attachments/assets/137d0cfa-27ee-4da5-969b-15596e232139)

**Comment by mengW6** [comment]
![image](https://github.com/user-attachments/assets/3900bb05-7ef9-4229-aa41-fca475d402ed)
When calculating cross entropy loss or classification tasks, the target label t exceeds the range of predicted categories (i.e., outside the number of categories n_classes), resulting in calculation failure. Do you know how to modify it? Thank you very much for your guidance.

**Comment by mengW6** [comment]
There was an error while running train_2d_transformar_v5. py. I checked and found that there was an error in calculating cross entropy due to pred shape: torch Size ([3136, 1027]), n_classes: 1027, but there are more than 1027 numbers in the target tag gold, but I couldn't find a solution. Can you give me some guidance?
![image](https://github.com/user-attachments/assets/fcb8085c-b2b8-4ab5-ab35-5eeb30b38769)
![image](https://github.com/user-attachments/assets/2cef5bce-91f3-4aa8-bafd-fa9215929da7)

**Comment by Garfield-kh** [comment]
Hi, I meet such error when preprocessing the data, the data is save in text format, if it is not deleted at all when re-do the preprocessing, it will continue writing in the text file, which case 188 569 together as 188569. 
So maybe you can 
1. check where the text data file, contain such large number. see if it is caused by re-writing bug.
2. delete all preprocessing file and re-do it.

hope this one fix~

**Comment by mengW6** [comment]
Thank you very much.
